test/lib/ufe/testComboCmd.py

```diff
@@ -805,6 +805,54 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
+    @unittest.skipIf(int(cmds.about(majorVersion=True)) > 2022, 'Center pivot command is only available in Maya Preview Release 124 or later.')
```

Comment:
We want this test to be run in master builds, not skipped in master builds. Additionally, we can leverage API version to detect update versions. Here is a check that will do this:
```suggestion
    @unittest.skipIf(int(cmds.about(apiVersion=True)) <= 20220000, 'Center pivot command is only available in Maya Preview Release 124 or later.')
```

We are still lacking coverage in PFs, this is why it wasn't detected when you run it.